site/docs/guide/files.md

```diff
@@ -159,7 +159,7 @@ On Deno, you can send `Blob` objects, too.
 ```ts
 // Send a buffer or a byte array.
 const buffer = Uint8Array.from([65, 66, 67]);
-new InputFile(buffer); // "ABC"
+new InputFile(buffer); // “ABC”
```

Comment:
These things are debatable. Using `"ABC"` would refer to the JavaScript value, using “ABC” would refer to the content as understood by humans. I'm okay with both ways, I just want to make sure that this was considered and a deliberate choice.

Reply:
I haven't touched the quotes referring to JavaScript values. For example in the quotes in [this line](https://github.com/grammyjs/website/blob/e89ca6901af89c09b921331fa2d62d6b7d1906a1/site/docs/guide/getting-started.md?plain=1#L57).

Reply:
I saw—not talking about that.

I do mean the comments. The old version was saying, _this buffer is equivalent to putting `"ABC"` into a file_, and the new version is saying, _if you look at the file it will contain the word “ABC”_. This is implied by using different quotes. Does this make more sense?

Reply:
I'll fix it.

Reply:
I'm not saying this is an isse, there are two options and I am fine with both. My point was rather that we both agree that these two options exist, and that we make an active choice, rather than performing these changes accidentally, since they are changing the meaning :)

Reply:
So just go ahead with whatever you prefer, be it changing things back or leaving them as-is.

Reply:
We'll use `"` if the contents inside the quotes are *only* referring to data. How's this?

Reply:
Sounds good! How do you manage to remember all these things? :smile: